_posts/2016-01-13-opendota-mmr-and-you.md

```diff
@@ -5,6 +5,10 @@ summary:    We've been getting a lot of questions about how MMR tracking works n
 categories: 
 ---
 
+**Update:** MMR sharing no longer works, the only data available at player's MMR page is the data that has been
```

Comment:
I'd also put the date of this update.

"data available on a player's MMR page"

"The instructions on this post are no longer valid."